client/src/ViewPhotosButton.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledButton = styled.button`
+  position: absolute;
+  margin: 29% 0% 0% 89%;
+  border-radius: 4px;
+  background-color: white;
+  color: #484848;
+  height: 32px;
+  width: 112px;
+`;
+
+const StyledDiv = styled.div`
```

Comment:
Have you considered moving your styled components for another folder? Separation of concerns sorta thing.